core/src/main/scala/akka/kafka/internal/ProducerStage.scala

```diff
@@ -9,7 +9,7 @@ import akka.kafka.ProducerMessage.{Message, Result}
 import akka.stream._
 import akka.stream.ActorAttributes.SupervisionStrategy
 import akka.stream.stage._
-import org.apache.kafka.clients.producer.{Callback, KafkaProducer, RecordMetadata}
+import org.apache.kafka.clients.producer.{Callback, Producer => KafkaProducer, RecordMetadata}
```

Comment:
Importing `Producer` but still calling it `KafkaProducer` seems confusing here - couldn't we just import `Producer`?

Reply:
changed in 564a2ad